Messenger/AppDelegate.mm

```diff
@@ -114,6 +114,10 @@ - (void)mouseExited:(NSEvent*)ev {
   _titlebarView.layer.opacity = 0;
 }
 
+- (IBAction)find:(NSMenuItem *)sender {
+  [_webView.windowScriptObject evaluateWebScript:@"document.querySelector('input[type=text]').focus();"];
```

Comment:
You could use something like this instead: `document.querySelector('input[placeholder~="Search"]')`